packages/screens/AdministrationDashboard/component/ApplicationTable.tsx

```diff
@@ -0,0 +1,152 @@
+import moment from "moment";
+import React from "react";
+import { FlatList, View } from "react-native";
+
+import { CollectionNameCell } from "../../../components/applicationTable/CollectionNameCell";
+import { InnerCellText } from "../../../components/applicationTable/InnerCellText";
+import { LinkIconAndRedirect } from "../../../components/applicationTable/LinkIconAndRedirect";
+import { TableRow } from "../../../components/table/TableRow";
+import { useIsMobile } from "../../../hooks/useIsMobile";
+import { mineShaftColor } from "../../../utils/style/colors";
+import {
+  layout,
+  screenContentMaxWidthLarge,
+} from "../../../utils/style/layout";
+
+const TABLE_ROWS = {
+  rank: {
+    label: "#",
+    flex: 1,
+  },
+  collectionNameData: {
+    label: "Collection Name",
+    flex: 5,
+  },
+  collectionNetwork: {
+    label: "Collection Network",
+    flex: 3,
+  },
+  TwitterURL: {
+    label: "Twitter URL",
+    flex: 2,
+  },
+  DiscordURL: {
+    label: "Discord URL",
+    flex: 2,
+  },
+  expectedTotalSupply: {
+    label: "Expected Total Supply",
+    flex: 3,
+  },
+  expectedPublicMintPrice: {
+    label: "Expected Public Mint Price",
+    flex: 3,
+  },
+  expectedMintDate: {
+    label: "Expected Mint Date",
+    flex: 3,
+  },
+};
+
+export const ApplicationTable: React.FC<{
+  rows: any[]; // currently i don't know the data types will change it once i will work on functionality
+}> = ({ rows }) => {
+  const isMobile = useIsMobile();
+
+  return (
+    <View
+      style={{
+        justifyContent: "space-between",
+        width: "100%",
+        maxWidth: screenContentMaxWidthLarge,
+      }}
+    >
+      <TableRow
+        headings={
+          !isMobile
+            ? Object.values(TABLE_ROWS)
+            : Object.values(TABLE_ROWS).slice(0, -5)
+        }
+      />
+
+      <FlatList
+        data={rows}
+        renderItem={({ item, index }) => <ApplicationRowData rowData={item} />}
+        keyExtractor={(item) => item.id}
+        style={{
+          minHeight: 220,
+          borderTopColor: mineShaftColor,
+          borderTopWidth: 1,
+        }}
+      />
+    </View>
+  );
+};
+
+const ApplicationRowData: React.FC<{ rowData: any }> = ({ rowData }) => {
```

Comment:
What do you think about renaming "row data" as "row item" ?
I think it's more clear, because "data" could be everything, but "item" involves a notion of hierarchy (A set of items). Ex:  `FlatList renderItem={({ item }) =>`

(Components, types and variables names)